python/cudf_polars/cudf_polars/experimental/distinct.py

```diff
@@ -157,6 +151,24 @@ def _(
     # Extract child partitioning
     child, partition_info = rec(ir.children[0])
     try:
-        return lower_distinct(ir, child, partition_info, rec.state["config_options"])
+        config_options = rec.state["config_options"]
```

Comment:
Can we tighten up the `try / except`, mainly as a future-proofing measure? IIUC, the only thing that should possibly throw the `NotImplementedError` is `lower_distinct`.